README.md

```diff
@@ -0,0 +1,50 @@
+# NEB Tool
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 16.2.8.
+
+## Dependencies
+
+- We are using NodeJS v 20.9 LTS [nodejs.org](https://nodejs.org/en/download)
+
+- To install all required packages: `npm install`
+
+- To install the Angular CLI which is required for tests, `npm install -g @angular/cli`
+
+    - Note: If you don't want to install angular cli globally, you can install it locally using `npm install @angular/cli` and then run it using `npm run-script ng`
+
+## For Developers
+
+- When developing in electron window use `npm run build-watch` and a re-build will trigger on save of changes
+
+- To start the electron app (kill and restart app after rebuild on save): `npm run electron`
+
+- When developing for web run `npm run start` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+
+## Build
+
+- Built artifacts will be stored in the `/dist` directory.
+
+- General build for electron `npm run build`
+
+- Production Web Build `npm run build-prod`
+
+- Production Electron Build `npm run build-prod-electron`
+
+## Native Installers
+
+- `npm run dist` will create electron installers for your operating system
+
+- Installer will be created in an `./output/neb-tool/` directory 
+
```

Comment:
Right now the tests do not run without Google chrome installed. It looks like there are configuration files for Karma that allow users to specify which browsers to test on: https://karma-runner.github.io/6.4/config/configuration-file.html

I think we should either include these files and instructions on how to set up the test environment, or add some note about it here in the readme.

Reply:
Updated README with note on chrome dependency and link to Karma config docs.